.github/workflows/build.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           path: ${{ github.workspace }}/packages/*.tar.gz
 
   macos-build:
-    runs-on: macos-10.15
+    runs-on: macos-11
```

Comment:
The `macos-10.15` is getting deprecated and is currently failing (the runner doesn't' start).